pkg/allocate/allocate.go

```diff
@@ -129,7 +136,7 @@ func IterateForAssignment(ipnet net.IPNet, rangeStart net.IP, rangeEnd net.IP, r
 			continue
 		}
 		// Assign and reserve the IP and return.
-		logging.Debugf("Reserving IP: |%v|", ip.String()+" "+containerID)
+		logging.Debugf("Reserving IP: |%v|", ip.String()+" "+podRef+" "+containerID)
```

Comment:
```suggestion
logging.Debugf("Reserving IP: |%s %s %s|", ip.String(), podRef, containerID)
```
... wondering if that yields the same output